doc/docs.md

```diff
@@ -3841,6 +3843,17 @@ for example `main`, you can use: `-printfn main -o file.c`.
 To see a detailed list of all flags that V supports,
 use `v help`, `v help build` and `v help build-c`.
 
+**Visual debugging Setup:**
+* [Visual Studio Code](vscode.md)
+
+### Javascript Backend
+
+There is currently no support for sourcemaps.
+
+### WASM Backend
+
+Not tested. Might work by using `lldb` and a WASM runtime like `wasmtime`.
+
```

Comment:
Unlike the others, there is not a concrete `WASM Backend` yet. 

WASM generation is supported for the moment only indirectly, through compiling with emscripten, for example:
```sh
v -d emscripten -o 2048.c examples/2048/
emcc -fPIC -w   2048.c   thirdparty/stb_image/stbi.c   -Ithirdparty/stb_image -Ithirdparty/fontstash -Ithirdparty/sokol -Ithirdparty/sokol/util   -o examples/2048/index.js   -DSOKOL_GLES2 -DSOKOL_NO_ENTRY   -DNDEBUG -O3   -s ERROR_ON_UNDEFINED_SYMBOLS=0 -s ALLOW_MEMORY_GROWTH -s MODULARIZE -s ASSERTIONS=1  --embed-file ./examples/assets/fonts/RobotoMono-Regular.ttf@/assets/fonts/RobotoMono-Regular.ttf
cd examples/2048/
wget https://v2048.vercel.app/index.html
python -m SimpleHTTPServer
firefox index.html
```

Reply:
This is an old example - the latest version of this repo does not use emscript.
I build a vs code remote container setup based on it to test the process but could not get debugging working.
The included Dockerfile should work without VS Code and allows to experiment without installing every lib on the local system
https://github.com/aheissenberger/vlang-wasm-example

Build Script:
https://github.com/aheissenberger/vlang-wasm-example/blob/main/build_wasm.sh